tests/integration/components/hyper-table-v2/column-test.ts

```diff
@@ -46,4 +49,38 @@ module('Integration | Component | hyper-table-v2/column', function (hooks) {
 
     assert.dom('.hypertable__column div.yielded').hasText('foobar');
   });
+
+  test('it looks up the rendering component for the column header', async function (assert: Assert) {
+    const renderingResolverSpy = sinon.spy(this.handler.renderingResolver);
+
+    await render(hbs`<HyperTableV2::Column @handler={{this.handler}} @column={{this.column}} />`);
+
+    // @ts-ignore
```

Comment:
What is the problem with the TS here?

Reply:
Sinon spy doesn't embed the original class' methods. looks like others have the problem but i'll keep looking into it to remove them